data/synapse-data-db2/src/main/java/io/americanexpress/synapse/data/db2/entity/BaseDb2Entity.java

```diff
@@ -23,25 +22,4 @@
 @MappedSuperclass
 public abstract class BaseDb2Entity {
 
-    /**
-     * The id of the entity.
-     */
-    @Id
-    private Long id;
```

Comment:
Synapse architecture shouldn't change for a use case or a POD system. That would compromise other apps architecture and design. I would recommend an alternative to change synapse.